lms/djangoapps/ccx/views.py

```diff
@@ -196,11 +196,20 @@ def dashboard(request, course, ccx=None):
     context = {
         'course': course,
         'ccx': ccx,
-        'STATE_CHOICES': STATE_CHOICES
+        'STATE_CHOICES': STATE_CHOICES,
+        'is_instructor': has_access(request.user, 'instructor', course),
+        'is_ccx_coach': False
     }
     context.update(get_ccx_creation_dict(course))
     if ccx:
         context.update(edit_ccx_context(course, ccx, request.user))
+        ccx_locator = CCXLocator.from_course_locator(course.id, unicode(ccx.original_ccx_id))
+        custom_courses = CustomCourseForEdX.objects.filter(course_id=ccx_locator)
```

Comment:
Regarding the first comment: we actually need only coaches on ccx's, not the master course.

Regarding the second comment: yes... when they ask us to do it :smile: